pkg/operator/config-reloader.go

```diff
@@ -129,6 +129,13 @@ func Shard(shard int32) ReloaderOption {
 	}
 }
 
+// ImagePullPolicy sets the imagePullPolicy option for the config-reloader container
+func ImagePullPolicy(imagePullPolicy v1.PullPolicy) ReloaderOption {
+	return func(c *ConfigReloader) {
+		c.config.ImagePullPolicy = imagePullPolicy
```

Comment:
imagePullPolicy needs to be a field of ConfigReloader (consistent with Shard() and other functions).

```suggestion
		c.imagePullPolicy = imagePullPolicy
```